### PR TITLE
Pgpro13

### PR DESCRIPTION
**Pull request by dlepikhova**
The proposed patch allows testing this extension in installcheck mode.

### CODE REVIEW

File: expected/corner_cases_3.out [hunk 1]
```diff
@@ -0,0 +1,67 @@
+Parsed test spec with 2 sessions
```
Comment (ololobus) [inline expected/corner_cases_3.out:1]: Can you, please, add info into review section of README (https://github.com/postgrespro/pg_query_state#tests) about Postgres/OS versions where we need this new variants of test outputs (corner_cases_3.out, corner_cases_4.out)?

**Review by ololobus — COMMENTED** [review]
Otherwise it looks good to me, if everything compiles and works well with all supported version

File: expected/corner_cases_3.out (hunk 1 above)
Comment (dlepikhova) [reply expected/corner_cases_3.out:1]: Yes, I updated README.md and added this information (in commit a586969).

**Comment by ololobus** [comment]
This PR seems to be outdated and should be replaced with a new PR from https://github.com/postgrespro/pg_query_state/tree/PGPRO13_installcheck, IIUC. @dlepikhova?

**Pull request closed without merging**